### ISSUE TITLE
Application node has no children if cluster has no service catalog enabled

### ISSUE THREAD

**Issue by mohitsuman** [redhat-developer/vscode-openshift-tools#137]
The ServiceInstance error occurs when no service catalog is enabled on the cluster.
 Error trace example:
``` 
oc get ServiceInstance -o jsonpath="{range .items[?(.metadata.labels.app == 'qwer')]}{.metadata.labels.app\.kubernetes\.io/component-name}{'\n'}{end}" --namespace try

error: the server doesn't have a resource type "ServiceInstance"
```

We need to add a condition to check that and then run the command for it.

**Comment by dgolovin** [comment]
It still supposed to work, just error message when explorer builds application node children list.

**Comment by jrichter1** [comment]
The UI should at least report what the problem is in this instance. Right now I can only see the stdout from odo and it doesn't feel at all intuitive.

**Comment by dgolovin** [comment]
Fixed via 5fac2061eb32c2a10a0b90028acf54c3625c5604.
